part-configuration.tex

```diff
@@ -187,20 +187,245 @@ \section{Matching your business processes}
+    \item[Tax] This checkmark identifies the account as a Tax (VAT)\index{tax} account. Tax accounts need
+    to be further configured. See \charef{cha-taxes} for further discussion of the
+    subject.
+    \item[Obsolete] This checkmark identifies the account as not being usable going forward.  The account cannot be deleted because it may
+    contain data.
+    \label{item:AccountOptionsTax}
+\end{description}
+
+
+\subsubsection{Include in drop-down menus}
+\label{subsec-coa-account-links}
+
+In order to enhance the user interface accounts may be added to drop down menus throughout LedgerSMB.
+
+\subsubsection{Receivables checkmarks}
+\label{subsubsec-coa-AR-checkmarks}
+
+\begin{description}[style=nextline]
+    \item[Income (AR\_amount)] This check mark adds the account to the list of accounts
+    in the transaction and invoice screens which are used to post income on.
+    \item[Payment (AR\_paid)] This check mark adds the account to the list of accounts
+    to choose from in the Receipts (AR) and Payments (AP) screens. Additionally, it
+    adds the account to the part entry screen as described in \secref{subsec-products-parts-definition}.
+    \item[Tax (AR\_tax)] This check mark makes the account show up as a check mark on the
+    \gls{customer} (AR) or vendor (AP) entry screen. See \charef{cha-taxes} for further discussion.
+    \item[Overpayment (AR\_overpayment)] Adds the account to the receipts screen as discussed
+    in \secref{sec-business-processes-payment-processing-overpayments}.
+    \item[Discount (AR\_discount)] Adds the account to the customer entry screen's selection
+    list for accounts to post 
+\end{description}
+
+\subsubsection{Payables checkmarks}
+\label{subsubsec-coa-AP-checkmarks}
+
+The payables UI works the same way as the receivables described in the previous list. The difference is
+that the technical names of the configuration identifiers are prefixed by AP\_ instead
+of AR\_.
+
+\subsubsection{Tracking Items}
+\label{subsubsec-coa-tracking-items}
+
+The items on this line relate to stocked items, i.e. those tracked for inventory: parts and
+assemblies.
+
+\begin{description}[style=nextline]
+    \item[Income (IC\_sale)] Adds the account to the selection list of income accounts on the
+    part and assembly definition screens.
+    \item[COGS (IC\_cogs)] Adds the account to the selection list of COGS @@@ accounts on the
+    part, assembly and overhead definition screen.
+    \item[Tax (IC\_taxpart)] Adds a check mark to the part and assembly definition screen
+    for the applicable account. See \charef{cha-taxes} for more details on how taxes
+    work in LedgerSMB.
+    \item[Returns (@@@ Need help here)] @@@
+\end{description}
+
+@@@ Question: Labor/Overhead accounts == inventory accounts??  These notes were in the original file and I don't know if they are still applicable.
+
+@@@ Labor/Overhead has an inventory and expense account but no income account. Chris T

```

Comment:
Labor sold isn't "inventory tracked", so is a service. Labor/Overhead used to manufacture assemblies *is* "inventory tracked" and capitalized. We should explain somewhere that Labor sold immediately is a service instead.

Reply:
I punted on this and just put your comment in the book for now.  It requires more thought and probably content additions.